lib/smb2/packet/close_request.rb

```diff
@@ -9,6 +9,11 @@ class CloseRequest < Smb2::Packet
     unsigned :flags, 16
     unsigned :reserved, 32
     string :file_id, 128
+
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Unnecessary if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)